### ISSUE TITLE
Exception: decodeObjectForKey: class "NFFieldNotification" not loaded or does not exist 

### ISSUE THREAD

**Issue by yuenpinkie** [underwindfall/NFCAndroid#3]
I have built the iOS tag reader app. After I read the android HCE Card, I have encountered this issue. My developer account is valid so what's the problem with this? Thank you.

**Comment by yuenpinkie** [comment]
> I have built the iOS tag reader app. After I read the android HCE Card, I have encountered this issue. My developer account is valid so what's the problem with this? Thank you.

FYI I am using iOS13 with iPhone 8 Plus and Galaxy S7 as android hce

**Comment by underwindfall** [comment]
@yuenpinkie It's same as issue #1 .  You may find more details here.

**Comment by yuenpinkie** [comment]
> @yuenpinkie It's same as issue #1 . You may find more details here.

Thanks for replying first of all. I have read the issue you quoted to me and I have checked the developer account. I found the developer account is valid with NFC Tag reading capability. So I have the same thoughts with user ulian-onua that the problem may due to the android HCE...as I have also used another NFC reading app on App Store to test out your android HCE project.
